File: 0001 Code Chef/0020 Masterchief final/implementation/0005 kotlin/MasterChef.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var test:Int? = stringInput.toIntOrNull();
        if(test != null){
            var count:Int = test;
            while(count > 0){
                var stringInput:String? = readLine();
                if(stringInput != null){
                    var rank:Int? = stringInput.toIntOrNull();
                    if(rank != null){
                        if(rank <= 10){
                            println("YES");
                        }else{
                            println("NO");
                        }
                    }
                }
                count = count - 1;
            }
        }
    }
}

File: 0001 Code Chef/0021 Masterchief final/implementation/0003 c#/MasterChef.cs
using System;

class MasterChef{
    static void Main(string[] args){
        string stringInput;
        stringInput = Console.ReadLine();
        int test = Convert.ToInt32(stringInput);
        while(test > 0){
            stringInput = Console.ReadLine();
            int rank = Convert.ToInt32(stringInput);
            if(rank <= 10){
                Console.WriteLine("YES");
            }else{
                Console.WriteLine("NO");
            }
            test = test - 1;
        }
    }
}